src/TimeSeries.ts

```diff
@@ -77,13 +77,14 @@ export interface TimeseriesSearchParams {
 const timeSeriesUrl = (): string => `${apiUrl(0.5)}/${projectUrl()}/timeseries`;
 
 export class TimeSeries {
-  public static async create(timeSeries: Partial<Timeseries>[]): Promise<void> {
+  public static async create(timeSeries: Partial<Timeseries>[]): Promise<Timeseries[]> {
     const body = {
       items: timeSeries,
     };
     const url = timeSeriesUrl();
-    await rawPost(url, { data: body });
-    return;
+    const response = (await rawPost(url, { data: body })) as AxiosResponse<
+    TimeseriesResponse>;
+    return response.data.data.items;
```

Comment:
This will throw? `response.data === {}` https://doc.cognitedata.com/api/0.5/#operation/postTimeSeries
Please see https://github.com/cognitedata/cognitesdk-js/issues/186#issuecomment-507727041

Reply:
You're right!